Classes/Domain/Search/ResultSet/Result/SearchResult.php

```diff
@@ -35,6 +35,24 @@
  */
 class SearchResult extends Document
 {
+    /**
+     * The variant field value
+     *
+     * Value of "variants.variantField"
```

Comment:
```suggestion
     * Value of "variants.variantField" of Solrs response.
```

We have the same TypoScript setting https://docs.typo3.org/p/apache-solr-for-typo3/solr/master/en-us/Configuration/Reference/TxSolrSearch.html#variants-variantfield. 
Would propose at least a proper comment, to avoid double Dutch on the contexts.
Better is the speaking name.

Reply:
This comment refers to the TS setting, not to the Solr response.

`variantFieldValue ` will be filled with the content of the field configured in TS setting `variants.variantField`

Reply:
ok, I see, this comment mistakable, I will change this to:

`Value of Solr collapse field, which is defined via TypoScript variable "variants.variantField"`

Reply:
Done